src/main/java/com/kodilla/ecommercee/domain/OrderProduct.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity(name = "ORDER_PRODUCTS")
+public class OrderProduct {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "ID", unique = true)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "ORDER_ID")
+    private Order order;
+    @ManyToOne
+    @JoinColumn(name = "PRODUCT_ID")
+    private Product product;
+
+    private BigDecimal price;
+
+    private Long qty=0L;
```

Comment:
I think the value 0L is unnecessary, it's going to be set in the constructor anyway, so i'd just do private Long qty;